test/jobs/purge_job_test.rb

```diff
@@ -40,15 +40,28 @@ def teardown
     assert_includes(email_body, 'Test StandardError')
   end
 
+  test 'sets monitoring, purged, and continuous_exposure' do
+    patient = create(:patient, monitoring: false, purged: false)
```

Comment:
Because both monitoring and continuous exposure are set to false here upon creation, this doesn't actually test that they were updated. I think if we just set them both to true then this would be sufficient.

Reply:
If either are true the patient won't be marked eligible for this job.

Reply:
Right, but we want to check that the job actually sets them both to false - that's what the test should be doing.